arrow-site/docs/docs/quickstart/README.md

```diff
@@ -7,28 +7,199 @@ permalink: /core/
-- [Effects]({{ '/effects/async/' | relative_url }}): interfacing with external systems
-- [Optics]({{ '/optics/iso/' | relative_url }}): inspecting and modifying data structures
+## [Fx]({{ '/fx/' | relative_url }})
+Arrow Fx is a full-featured, high-performance, asynchronous framework that brings functional operators to Kotlin's `suspend` functions.
+By leveraging the power of KotlinX Coroutines and the compiler support for CPS transformations, Arrow Fx results in optimal async programs with increased throughput and decreased allocations.
 
-## Curated external links
+## [Optics]({{ '/optics/' | relative_url }})
+Arrow Optics provides an automatic DSL that allows users to use `.` notation when accessing, composing and transforming deeply nested immutable data structures.
+Optics offers additionally all the base types such as [Lens]({{ '/optics/lens/' | relative_url }}), [Prism]({{ '/optics/prism/' | relative_url }}) and others from which we can generalize accessing and traversing deep values in sealed and data classes models.
 
-- [Projects and Examples]({{ '/quickstart/projects/' | relative_url }})
-- [Media](https://media.arrow-kt.io)
+## [Meta]({{ '/meta/' | relative_url }})
+Arrow Meta is a general purpose library for meta-programming in Kotlin to build compiler plugins.
+Some type system features proposed by Arrow such as union types, product types, proof derivation and others are built with Arrow Meta and serve as exploration of what could be incorporated in the Kotlin compiler.
 
 ## Setup
 
-Take a look at [Setup]({{ '/quickstart/setup/' | relative_url }}) where you'll find the instructions to configure a project with `Gradle` or `Maven` to use `Arrow` library.
+### Next development version
+
+If you want to try the last features, replace `0.13.0` by `1.0.0-SNAPSHOT` in the following guideline.
+
+### JDK
+
+Make sure to have the latest version of JDK 1.8 installed.
+
+### Android
+
+Arrow supports Android starting on API 21 and up.
+
+### Gradle
+
+#### Basic Setup
+
+In your project's root `build.gradle`, append these repositories to your list:
+
+```groovy
+allprojects {
+    repositories {
+        mavenCentral()
+        maven { url "https://oss.jfrog.org/artifactory/oss-snapshot-local/" } // for SNAPSHOT builds
+    }
+}
+```
+
+Add the dependencies into the project's `build.gradle`:
+
+##### Arrow Core
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+def arrow_version = "0.13.0"
+dependencies {
+    implementation "io.arrow-kt:arrow-core:$arrow_version"
+}
+```
+
+##### Arrow Core + Arrow Optics
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+def arrow_version = "0.13.0"
+dependencies {
+    implementation "io.arrow-kt:arrow-optics:$arrow_version"
+    kapt    "io.arrow-kt:arrow-meta:$arrow_version"
+}
+```
+
+##### Arrow Core + Arrow Fx
+
+```groovy
+def arrow_version = "0.13.0"
+dependencies {
+    implementation "io.arrow-kt:arrow-fx-coroutines:$arrow_version"
+}
+```
```

Comment:
Can we combine this into a single block? I think it's as clear but only a third so long.

```groovy
apply plugin: 'kotlin-kapt'
def arrow_version = "0.13.0"
dependencies {
     // Arrow Core
    implementation "io.arrow-kt:arrow-core:$arrow_version"
    
    // Arrow Fx
    implementation "io.arrow-kt:arrow-fx-coroutines:$arrow_version"
    
    // Arrow Optics
    implementation "io.arrow-kt:arrow-optics:$arrow_version"
    kapt    "io.arrow-kt:arrow-meta:$arrow_version"
}
```

Reply:
In the past people would complain and ask in the channel, what to do if they solely need Core or Fx or Optics, that's why we split that.

Reply:
The single block's concern is that most users seem to copy and paste everything they find on this page. I propose we do another pass to this page before 1.0 in a different PR to address the dependencies section and how they are presented.